src/rcms/fm/app/level1/HCALEventHandler.java

```diff
@@ -1760,39 +1612,39 @@ protected Boolean isTriggerAdapterStopped() {
             if(!functionManager.HBHEain) {
               if (functionManager.FMrole.equals("HCAL")) {
-                logger.warn("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HBHEa FED range.\nEnabling the HBHEa partition.");
+                logger.info("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HBHEa FED range.\nEnabling the HBHEa partition.");
               }
               functionManager.HBHEain = true;
             }
           }
           else if ( FedId >= functionManager.firstHBHEbFedId && FedId <= functionManager.lastHBHEbFedId ) {
             if(!functionManager.HBHEbin) {
               if (functionManager.FMrole.equals("HCAL")) {
-                logger.warn("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HBHEb FED range.\nEnabling the HBHEb partition.");
+                logger.info("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HBHEb FED range.\nEnabling the HBHEb partition.");
               }
               functionManager.HBHEbin = true;
             }
           }
           else if ( FedId >= functionManager.firstHBHEcFedId && FedId <= functionManager.lastHBHEcFedId ) {
             if(!functionManager.HBHEcin) {
               if (functionManager.FMrole.equals("HCAL")) {
-                logger.warn("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HBHEc FED range.\nEnabling the HBHEc partition.");
+                logger.info("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HBHEc FED range.\nEnabling the HBHEc partition.");
               }
               functionManager.HBHEcin = true;
             }
           }
           else if ( FedId >= functionManager.firstHFFedId && FedId <= functionManager.lastHFFedId ) {
             if(!functionManager.HFin) {
               if (functionManager.FMrole.equals("HCAL")) {
-                logger.warn("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HF FED range.\nEnabling the HF partition.");
+                logger.info("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HF FED range.\nEnabling the HF partition.");
               }
               functionManager.HFin = true;
             }
           }
           else if ( FedId >= functionManager.firstHOFedId && FedId <= functionManager.lastHOFedId ) {
             if(!functionManager.HOin) {
               if (functionManager.FMrole.equals("HCAL")) {
-                logger.warn("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HF FED range.\nEnabling the HO partition.");
+                logger.info("[HCAL " + functionManager.FMname + "] FedId = " + FedId + " is in the HCAL HF FED range.\nEnabling the HO partition.");
               }
               functionManager.HOin = true;
```

Comment:
All of these `functionManager.HOin` etc are not used anywhere else in the code and is largely superseded by `EMPTY_FM` parameter. 
Besides, the FedId range change still uses VME FED ranges. We could remove 
the section below `// check if FEDs from a specific HCAL partition are enabled`

Reply:
Let's leave that particular cleanup job for a second phase of cleaning.

Reply:
ok